gh-pages/_docs/06-02-user-controls.md

```diff
@@ -15,7 +15,7 @@ The controls that are available to you are:
 | #   | Type            | Action                                                                                                                                                                                                                                                           |
 | --- | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | 1   | Settings        | Load cc.json map or multiple maps (_hold shift_).                                                                                                                                                                                                                |
-| 2   | Settings        | Download the current map as a JSON-file. **Deprecation Warning!** This feature will soon be removed. It is already replaced by the controls nr. 15 and 16.                                                                                                       |
+| 2   | Settings        | **Removed!** This button no longer exists and has been replaced by the controls Nr. 15 and 16.                                                                                                                                                                   |
```

Comment:
Let's remove this line completely :-)
```suggestion
```

Reply:
Ok, I see that there is a gap then. So it is fine for me to keep it until we update the screenshot.